components/Controllers/Loans/VaultHealth.tsx

```diff
@@ -1,9 +1,10 @@
+import { HealthBar } from 'components/HealthBar';
 import { Tooltip } from 'components/Tooltip';
 import { ethers } from 'ethers';
 import { useController } from 'hooks/useController';
 import { convertOneScaledValue } from 'lib/controllers';
 import { formatPercent } from 'lib/numberFormat';
-import React, { useMemo } from 'react';
+import React from 'react';
```

Comment:
do we need to import React here?

Reply:
It isn't strictly required but also not an error